MediaBrowser.WebDashboard/dashboard-ui/strings/ar.json

```diff
@@ -1010,6 +1010,12 @@
   "LabelValue": "القيمة:",
   "LabelVersionInstalled": "{0} مثبتة",
   "LabelVersionNumber": "الإصدار {0}",
```

Comment:
This line seems to lack {

Reply:
It's at the back. My editor can't edit it properly since it's RTL. But only half, due to the brackets not being Arabic. And it displays correctly here in Firefox, but not in my editor. Ooh Unicode.